.travis.yml

```diff
@@ -59,6 +59,7 @@ cache:
   - $BOOST_ROOT
   - llvm-$LLVM_VERSION
   - cmake
+  - $GDB_ROOT
 
 before_install:
   - bin/ci/ubuntu/install-dependencies.sh
```

Comment:
Yes, it does. (I thought I did that, but I only did it for circle ci, not travis). Thanks @mellery451. Will fix